per/admin.py

```diff
@@ -155,10 +156,36 @@ class FormComponentQuestionAndAnswerAdmin(admin.ModelAdmin):
     pass
 
 
-class OpsLearningAdmin(admin.ModelAdmin):
+class OpsLearningAdmin(admin.ModelAdmin, GotoNextAdminMixin):
+
+    def response_change(self, request, obj):
+        """Determines the HttpResponse for the change_view stage."""
+        if '_save_next' in request.POST:
+            next_pk = self.get_next_instance_pk(request, obj)
+            if next_pk:
+                response = redirect('admin:per_opslearning_change', next_pk)
+                qs = request.GET.urlencode()  # keeps _changelist_filters
+            else:
+                # Last item (or no longer in list) - go back to list in the same position
+                response = redirect('admin:per_opslearning_changelist')
+                qs = request.GET.get('_changelist_filters')
+
+            if qs:
+                response['Location'] += '?' + qs
+            return response
+
+        return super().response_change(request, obj)
+# stackoverflow.com/questions/58014139/how-to-go-to-next-object-in-django-admin
+
+    def render_change_form(self, request, context, add=False, change=False, form_url='', obj=None):
+        context.update({'show_save_and_next': True})
+        return super().render_change_form(request, context, add, change, form_url, obj)
```

Comment:
We can move this to the Mixin as well.

For 'admin:per_opslearning_' we can use
```
        app_label = obj._meta.app_label
        model_name = obj._meta.model_name
        f'admin:{app_label}_{model_name}_change'
        f'admin:{app_label}_{model_name}_changelist'
```

Reply:
Hi @thenav56 , I couldn't make a working version by moving response_change to the mixin.

(Even not by leaving this in OpsLearningAdmin:  )
```
    def render_change_form(self, request, context, add=False, change=False, form_url='', obj=None):
        context.update({'show_save_and_next': True})
        return super().render_change_form(request, context, add, change, form_url, obj)
```